simpleflow/swf/process/actor.py

```diff
@@ -214,10 +214,10 @@ def start(self):
         self.set_process_name()
         while self.is_alive:
             try:
-                task = self._poll(self.task_list, self.identity)
+                response = self._poll(self.task_list, self.identity)
```

Comment:
It does try to check the type and rtype values, as well as to infer from the method code. For instance there's a warning in simpleflow/swf/executor.py:254 on the use of event: "Type 'TypeError' doesn't have expected attribute 'get'". That's since `find_event` may return a TypeError (guess it should raise?)